editor/editor_data.cpp

```diff
@@ -902,17 +902,11 @@ String EditorData::script_class_get_icon_path(const String &p_class) const {
 		return String();
 	}
 
-	String current = p_class;
-	String ret = _script_class_icon_paths[current];
-	while (ret.empty()) {
-		current = script_class_get_base(current);
-		if (!ScriptServer::is_global_class(current)) {
-			return String();
-		}
-		ret = _script_class_icon_paths.has(current) ? _script_class_icon_paths[current] : String();
+	if(_script_class_icon_paths.has(p_class)) {
```

Comment:
```suggestion
	if (_script_class_icon_paths.has(p_class)) {
```

See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html